CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.5.2 (2022-12-15)
+
+* Add "mjml_version" as optional parameter in configuration
```

Comment:
```suggestion
* Add "mjml_version" as an optional parameter in configuration
```

Also most likely this will be considered a feature and will go into 3.6.0 but that's up to maintainers